pages/contributing/style_guidelines.md

```diff
@@ -2,4 +2,89 @@
+* **-ise/-ize:** use the "-ise" form.
+* **Life cycle:** two separate words (not "lifecycle").
+* **Links:** make the link text say where the link goes, e.g. "the Contribute page", not "click here". Avoid using the url as the link text.
+* **Lists:** 
+   * _A list of short items_: introduce with a colon, start each bullet point with a capital and don't use punctuation at the end of each bullet point:
+      * Item 1
+      * Item 2
+    * _A list of longer items following an incomplete introductory sentence (e.g. a sentence ending in a colon)_: each item ends with a semi-colon and the final item ends with a full stop. Do not capitalise the first letter of each item, e.g. This is the first part of a sentence that includes:
+       * a longer item 1;
+       * a longer item 2;
+       * a longer item 3.
+    * _A list following a complete sentence (with a full stop)_: each item ends with a full stop, and each point begins with a capital letter, e.g. This a complete sentence.
+       * This is item 1 of the list.
+       * This is item 2 of the list.
+       * This is item 3 of the list.
+* **Numbers:** spell the numbers one to ten out. After that, write the numbers (11, 12, 13, etc.).
+* **Quotations:** use double quotes for quotations and single quotes for quotes within quotes.
+* **That/which:** use "that" when you are defining something and "which" when you are adding extra information about it e.g.:
+ * "The cat that was on the table suddenly got up" is telling us which cat it was. It is important to the meaning of the sentence because you are not talking about any cat, just the cat on the table.
+ * "The cat, which was sitting on the table, suddenly got up" is giving us extra information about the cat. The information is not necessary to understand the sentence. You can remove the clause, and the sentence will still be clear. Clauses starting with "which" usually begin with a comma.
+* **Titles (the "title" in the front matter of pages):** only the first word, proper nouns and acronyms are capitalised.
+* **Training:** training is an uncountable noun and cannot have a plural. You can write "training courses" and "training materials" but not "trainings".
+
+## Page graphics
+
+  * **White space:** make sure there is plenty of space so that the main elements stand out and the text does not appear overwhelming.
+  * **Colours:** To be decided
+  * **Fonts:** Exo 2 is used for headings and main branding font, Open Sans for body text.
+  * **Icons:** the icons used in the data life cycle diagram come from the [Noun Project](https://thenounproject.com/ELIXIRCommunications/kit/rdmkit/). We have a Pro license and the right to publish them without attribution. Other icons on this site are either desgined by Xènia Pérez Sitjà or come from [Font Awesome](https://fontawesome.com/).
```

Comment:
I think this needs to be changed - we don't have a pro license on EVERSE - but we don't use the data life cycle diagram - perhaps we can just remove this line

Reply:
Done.